.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "player_code"]
```

Comment:
- While updating .gitmodules, please clear the previous .git/modules and in .git/config